lib/fs.js

```diff
@@ -204,18 +204,31 @@ function validateOffsetLengthWrite(offset, length, byteLength) {
   }
 }
 
-function validatePath(path, propName) {
+function validatePath(path, options) {
```

Comment:
I think we are fine with something like `validatePath(path, propName = 'path', assertFilename = false)` here, there is no need to introduce the overhead of `getOptions`